### ISSUE TITLE
Crash of compose activity after closing contextual menu for recipient address

### ISSUE THREAD

**Issue by bbc2** [thunderbird/thunderbird-android#5309]
**To Reproduce**

- Compose email
- Enter valid recipient address, like `foo@example.org`
- Press on that email address until it is selected and the contextual menu appears.
- Tap anywhere outside of the menu to close it.

At this point, instead of just closing the menu, the compose activity is closed abruptly and the app shows me my inbox.

**Expected behavior**

The menu is closed and I can continue composing my email.

**Screenshots**

Just before the crash:

![screenshot](https://user-images.githubusercontent.com/1645335/119507815-73fe5780-bd6f-11eb-9797-3ba79a5c6868.png)

**Environment (please complete the following information):**
 - K-9 Mail version: 5.734
 - Android version: 11
 - Device: Google Pixel 4a
 - Account type: IMAP

**Logs**

```
05-25 15:24:48.561 26763 26763 I AssistStructure: Flattened final assist data: 4648 bytes, containing 1 windows, 28 views
05-25 15:24:50.478 26763 26763 D TokenAutoComplete: Current completion text: fo
05-25 15:24:50.650 26763 26763 D TokenAutoComplete: Current completion text: foo
05-25 15:24:52.181 26763 26763 D TokenAutoComplete: Current completion text: foo@
05-25 15:24:52.199 26763 26763 D TokenAutoComplete: Current completion text: foo@
05-25 15:24:52.649 26763 26763 D TokenAutoComplete: Current completion text: foo@e
05-25 15:24:52.929 26763 26763 D TokenAutoComplete: Current completion text: foo@ex
05-25 15:24:53.433 26763 26763 D TokenAutoComplete: Current completion text: foo@exa
05-25 15:24:53.649 26763 26763 D TokenAutoComplete: Current completion text: foo@exam
05-25 15:24:53.946 26763 26763 D TokenAutoComplete: Current completion text: foo@examp
05-25 15:24:54.186 26763 26763 D TokenAutoComplete: Current completion text: foo@exampl
05-25 15:24:54.537 26763 26763 D TokenAutoComplete: Current completion text: foo@example
05-25 15:24:54.901 26763 26763 D TokenAutoComplete: Current completion text: foo@example.
05-25 15:24:55.425 26763 26763 D TokenAutoComplete: Current completion text: foo@example.o
05-25 15:24:55.675 26763 26763 D TokenAutoComplete: Current completion text: foo@example.or
05-25 15:24:55.892 26763 26763 D TokenAutoComplete: Current completion text: foo@example.org
05-25 15:24:57.593 26763 26763 D androidtc: Initializing SystemTextClassifier, type = System
05-25 15:24:57.595 26763 26763 W androidtc: TextClassifier called on main thread
05-25 15:24:57.596 26763 26763 D androidtc: Initializing SystemTextClassifier, type = System
05-25 15:24:57.935 26763 26763 W androidtc: TextClassifier called on main thread
05-25 15:24:58.697 26763 26763 D TokenAutoComplete: Current completion text: foo@example.org
05-25 15:24:58.764 26763 26763 E InputEventReceiver: Exception dispatching input event.
05-25 15:24:58.764 26763 26763 E MessageQueue-JNI: Exception in MessageQueue callback: handleReceiveCallback
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: java.lang.IllegalArgumentException: Invalid offset: 16. Valid range is [0, 15]
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.method.WordIterator.checkOffsetIsValid(WordIterator.java:400)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.method.WordIterator.isBoundary(WordIterator.java:100)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.Editor$SelectionHandleView.positionAtCursorOffset(Editor.java:5922)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.Editor$HandleView.invalidate(Editor.java:4790)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.Editor$SelectionModifierCursorController.invalidateHandles(Editor.java:6772)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.Editor.invalidateHandlesAndActionMode(Editor.java:2157)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.TextView.setShadowLayer(TextView.java:4954)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.tokenautocomplete.TokenCompleteTextView.api16Invalidate(TokenCompleteTextView.java:562)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.tokenautocomplete.TokenCompleteTextView.invalidate(TokenCompleteTextView.java:571)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.TextView.updateAfterEdit(TextView.java:10733)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.TextView.handleTextChanged(TextView.java:10758)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13655)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.tokenautocomplete.TokenCompleteTextView.replaceText(TokenCompleteTextView.java:892)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.fsck.k9.view.RecipientSelectView.replaceText(RecipientSelectView.java:224)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.fsck.k9.view.RecipientSelectView.performCompletion(RecipientSelectView.java:254)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.fsck.k9.view.RecipientSelectView.onFocusChanged(RecipientSelectView.java:208)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.clearFocusInternal(View.java:7953)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.unFocus(View.java:7986)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1200)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1200)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:858)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.handleFocusGainInternal(View.java:7778)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.requestFocusNoSearch(View.java:13320)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.requestFocus(View.java:13294)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.requestFocus(View.java:13261)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.requestFocus(View.java:13203)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.onTouchEvent(View.java:15702)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.widget.TextView.onTouchEvent(TextView.java:11030)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.View.dispatchTouchEvent(View.java:14309)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:488)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1873)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4125)
05-25 15:24:58.765 26763 26763 E MessageQueue-JNI: 	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:446)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.View.dispatchPointerEvent(View.java:14568)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:6022)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5825)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5316)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5373)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5339)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5491)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5347)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5548)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5320)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5373)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5339)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5347)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5320)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:8086)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:8037)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7998)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:8209)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:220)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.os.MessageQueue.nativePollOnce(Native Method)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.os.MessageQueue.next(MessageQueue.java:335)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.os.Looper.loop(Looper.java:183)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at java.lang.reflect.Method.invoke(Native Method)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
05-25 15:24:58.766 26763 26763 E MessageQueue-JNI: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
05-25 15:24:58.766 26763 26763 D AndroidRuntime: Shutting down VM
--------- beginning of crash
05-25 15:24:58.766 26763 26763 E AndroidRuntime: FATAL EXCEPTION: main
05-25 15:24:58.766 26763 26763 E AndroidRuntime: Process: com.fsck.k9, PID: 26763
05-25 15:24:58.766 26763 26763 E AndroidRuntime: java.lang.IllegalArgumentException: Invalid offset: 16. Valid range is [0, 15]
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.method.WordIterator.checkOffsetIsValid(WordIterator.java:400)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.method.WordIterator.isBoundary(WordIterator.java:100)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.Editor$SelectionHandleView.positionAtCursorOffset(Editor.java:5922)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.Editor$HandleView.invalidate(Editor.java:4790)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.Editor$SelectionModifierCursorController.invalidateHandles(Editor.java:6772)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.Editor.invalidateHandlesAndActionMode(Editor.java:2157)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.TextView.setShadowLayer(TextView.java:4954)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.tokenautocomplete.TokenCompleteTextView.api16Invalidate(TokenCompleteTextView.java:562)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.tokenautocomplete.TokenCompleteTextView.invalidate(TokenCompleteTextView.java:571)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.TextView.updateAfterEdit(TextView.java:10733)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.TextView.handleTextChanged(TextView.java:10758)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.TextView$ChangeWatcher.onTextChanged(TextView.java:13655)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.SpannableStringBuilder.sendTextChanged(SpannableStringBuilder.java:1267)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:576)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:507)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:37)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.tokenautocomplete.TokenCompleteTextView.replaceText(TokenCompleteTextView.java:892)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.fsck.k9.view.RecipientSelectView.replaceText(RecipientSelectView.java:224)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.fsck.k9.view.RecipientSelectView.performCompletion(RecipientSelectView.java:254)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.fsck.k9.view.RecipientSelectView.onFocusChanged(RecipientSelectView.java:208)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.clearFocusInternal(View.java:7953)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.unFocus(View.java:7986)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1200)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1200)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:858)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.handleFocusGainInternal(View.java:7778)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.requestFocusNoSearch(View.java:13320)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.requestFocus(View.java:13294)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.requestFocus(View.java:13261)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.requestFocus(View.java:13203)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.onTouchEvent(View.java:15702)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.widget.TextView.onTouchEvent(TextView.java:11030)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.dispatchTouchEvent(View.java:14309)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3118)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2799)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:488)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1873)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4125)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:446)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14568)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:6022)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5825)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5316)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5373)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5339)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5491)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5347)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5548)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5320)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5373)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5339)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5347)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5320)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:8086)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:8037)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7998)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:8209)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:220)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:335)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:183)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
05-25 15:24:58.766 26763 26763 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
05-25 15:24:58.832 26763 26763 I Process : Sending signal. PID: 26763 SIG: 9
```

**Comment by cketti** [comment]
This is probably caused by the same TokenAutoComplete bug as #3951.

**Comment by cketti** [comment]
This is also fixed by #7958
